paddle/cinn/auto_schedule/tests/performance_comparison_test.cc

```diff
@@ -157,7 +157,7 @@ class PerformanceTester : public ::testing::Test {
 
     for (auto group : graph->fusion_groups) {
       compile_options.lowered_funcs.push_back(
-          op_lowerer->LowerWithoutSchedule(group));
+          op_lowerer->Lower(group, false, false));
```

Comment:
A tip in software engineering: boolean variables of functions have very bad readability. `Lower(group, false, false)`, what stands for the `false`? It happens when you have lots of `true` or `false`.

Usually, there are two ways to handle this problem.

1. Don't use boolean function
2. Add comment :`Lower(group, /*apply_op_schedule = */ false,  /*apply_group_schedule=*/ false)`
3. Use some parameters, such as:
```
    bool apply_op_schedule = false;
    bool apply_group_schedule = false;
    Lower(group, apply_op_schedule, apply_group_schedule);
```

Reply:
The call of the Lower() function has been modified using the second method.